src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/movements/MovementsResource.kt

```diff
@@ -55,4 +55,42 @@ class MovementsResource(
   fun getTemporaryAbsencesAndMovements(
     @Schema(description = "Offender number (NOMS ID)", example = "A1234BC") @PathVariable offenderNo: String,
   ): OffenderTemporaryAbsencesResponse = movementsService.getTemporaryAbsencesAndMovements(offenderNo)
+
+  @PreAuthorize("hasRole('ROLE_NOMIS_MOVEMENTS')")
+  @GetMapping("/movements/{offenderNo}/temporary-absences/application/{applicationId}")
```

Comment:
New endpoint that gets a movement application only - not any of the related schedules or movements themselves. This is because we're syncing fine grained entities, not families of entities.